app/scripts/lib/setupSentry.js

```diff
@@ -499,6 +504,11 @@ export default function setupSentry({ release, getState }) {
    * @returns `true` if MetaMetrics is enabled, `false` otherwise.
    */
   async function getMetaMetricsEnabled() {
+    // For MMI we have set MetaMetrics OFF by default, but want to keep Sentry logging
+    if (METAMASK_BUILD_TYPE === 'mmi') {
+      return true;
+    }
+
```

Comment:
`getMetaMetricsEnabled` returning `true` when it is _not_ enabled is a bit odd. Genuine question: how does this not just always enable MetaMetrics in MMI?

Reply:
Because `getMetaMetricsEnabled` is a method that just checks the MetaMetrics value from the app state in [here](https://github.com/MetaMask/metamask-extension/blob/0e3a442f406e9c5662417ce225f25f09bdbdd479/app/scripts/lib/setupSentry.js#L514) but since we are returning early when build type is MMI, it really doesn't matter. It serves the purpose so that Sentry starts it's session since [this](https://github.com/MetaMask/metamask-extension/blob/0e3a442f406e9c5662417ce225f25f09bdbdd479/app/scripts/lib/setupSentry.js#L586) will be true.

I think it's preferable to just add this `if` statement in the `getMetaMetricsEnabled` method instead of adding more code fences to the `startSession`, `endSession` and `toggleSession` in order to achieve the same result.